src/components/ShoppingCart/CartListItem.tsx

```diff
@@ -4,12 +4,17 @@ import { FaPen } from "react-icons/fa";
 import { MdDeleteForever } from "react-icons/md";
 import { ICartItem } from "@/interfaces/cartItem";
 
-const CartListItem = (props: ICartItem) => {
+interface CartListItemProps {
+  item: ICartItem;
+  onDelete: (id: string) => void;
+}
+
+const CartListItem = ({ item, onDelete }: CartListItemProps) => {
   // import cart information
```

Comment:
No need for comment. Coding should be self-documented themselves.

Reply:
Got it. These comments will not be reserved in the code.